### PR TITLE
use pnpm

### PR DESCRIPTION
**Pull request by patricklx**
## Description
now you can choose :)

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -193,14 +199,14 @@ jobs:
         env:
           PR: ${{ github.event.pull_request.number }}
       - name: Build
-        run: yarn ember build --environment production
+        run: pnpm ember build --environment production
       - name: Pack
         run: |
-          VERSION="$(jq -r '.version | split("+") | .[0]' package.json)"
```
Comment (RobbieTheWagner) [inline .github/workflows/build.yml:199]: Why was this changed?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -193,14 +199,14 @@ jobs:
...
+          VERSION="$(jq -r '.version' package.json)"
 
-          yarn pack
+          pnpm pack
           mkdir -p dist/npm
-          tar xvzf "ember-inspector-v$VERSION.tgz" -C dist/npm --strip-components 1
```
Comment (RobbieTheWagner) [inline .github/workflows/build.yml:203]: We probably want to keep this name the same, right?

**Review by RobbieTheWagner — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 2 above)
Comment (patricklx) [reply .github/workflows/build.yml:203]: it doesnt matter as its extracted anyway into dist/npm

Comment (patricklx) [reply .github/workflows/build.yml:203]: I misunderstood.
Pnpm pack generates a different filename

File: .github/workflows/build.yml (hunk 1 above)
Comment (RobbieTheWagner) [reply .github/workflows/build.yml:199]: So the new one will work with all possible version types? Not sure if this is also needed for nightly versions etc

File: .github/workflows/build.yml (hunk 2 above)
Comment (RobbieTheWagner) [reply .github/workflows/build.yml:203]: Okay, just makes me worried changing names

File: .github/workflows/build.yml (hunk 1 above)
Comment (patricklx) [reply .github/workflows/build.yml:199]: Well, the version string during the t

**Comment by patricklx** [comment]
I squashed all the changes into one commit

**Review by RobbieTheWagner — APPROVED** [review]
